hls4ml/templates/vivado/nnet_utils/nnet_conv1d_latency.h

```diff
@@ -84,5 +84,356 @@ void conv_1d_latency_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan],
     }
 }
 
+template <class data_T, class res_T, typename CONFIG_T>
+void pointwise_conv_1d_latency_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan / CONFIG_T::reuse_factor],
+                                  res_T res[CONFIG_T::out_width * CONFIG_T::n_filt / CONFIG_T::reuse_factor],
+                                  typename CONFIG_T::weight_t weights[CONFIG_T::n_chan * CONFIG_T::n_filt],
+                                  typename CONFIG_T::bias_t biases[CONFIG_T::n_filt]) {
+    assert(CONFIG_T::filt_width == 1);
```

Comment:
Do you think we could just remove this function, and then call dense latency instead????

Reply:
@Duchstf you're right it should be possible. i *think* it would require putting some of the logic in the optimizer pass because we'd need to create a new config for the dense call (and i don't think we want to create that config on the fly in the c++ side)